src/utils/saveToFile.js

```diff
@@ -5,7 +5,7 @@ const {logDebug} = require('./logger');
 
 module.exports = curry(async (fileName, data) => {
   try {
-    fsExtra.outputFileSync(fileName, JSON.stringify(await data, null, 2), 'utf8');
+    fsExtra.outputFileSync(fileName, JSON.stringify(await data), 'utf8');
```

Comment:
**question:** Why should we await on data here?

Reply:
Addressed by removing the redundant awaits